### DISCUSSION TITLE
Optimization numbers output to terminal

### DISCUSSION THREAD

**Discussion by lebsral** [stanfordnlp/dspy#1077] [Q&A]
<img width="1424" alt="image" src="https://github.com/stanfordnlp/dspy/assets/2120007/364db30a-f15a-4a55-85aa-459b613936b6">

Is there a place that explains, what these numbers mean, in detail? and how we should be using them?

**Comment by tom-doerr** [comment]
From left to right: Sum of scores / number samples to evaluate / average score percentage
What it means completely depends on what your metric is
